pkg/k8s/apis/cilium.io/v2/validator/validator.go

```diff
@@ -105,10 +113,91 @@ func (n *NPValidator) ValidateCNP(cnp *unstructured.Unstructured) error {
 	return nil
 }
 
+const (
+	warnEmptyToFromEndpointMessage = "It seems you have a CiliumClusterwideNetworkPolicy " +
+		"with an empty to/from endpoint selector. The behavior of this selector has been " +
+		"changed. The selector now only allows traffic to/from Cilium managed K8s endpoints, " +
+		"instead of acting as a truly empty endpoint selector allowing all traffic. To " +
+		"ensure that the policy behavior does not affect your workloads, consider adding " +
+		"another policy that allows traffic to/from world and cluster entities. For a more " +
+		"detailed discussion on the topic, see GitHub issue #12844"
+)
+
 // ValidateCCNP validates the given CCNP accordingly the CCNP validation schema.
 func (n *NPValidator) ValidateCCNP(ccnp *unstructured.Unstructured) error {
 	if errs := validation.ValidateCustomResource(nil, &ccnp, n.ccnpValidator); len(errs) > 0 {
 		return errs.ToAggregate()
 	}
+
+	logger := n.logger.WithFields(logrus.Fields{
+		logfields.CiliumClusterwideNetworkPolicyName: ccnp.GetName(),
+	})
+
+	// At this point we have validated the custom resource with the new CRV.
+	// We can try converting it to the new CCNP type.
+	// This should not fail so we are ignoring any errors here.
```

Comment:
This comment says that we are ignoring errors, but there's a else-clause that logs the error. I wouldn't consider that ignoring :wink:

Reply:
Ah, we are ignoring the error actually as we only log a warning instead of returning the error. But yeah, the comment is not clear. I will update it.